glue/sales_val_flagging.py

```diff
@@ -193,7 +232,18 @@ def group_size_adjustment(df, stat_groups: list, min_threshold, condos: bool):
         lambda row: "_".join([str(row[col]) for col in stat_groups]), axis=1
     )
 
-    return df_flagged_updated
+    df = df_flagged_updated.assign(
+        # PTAX-203 binary
+        sv_is_ptax_outlier=lambda df: df["sv_ind_ptax_flag_w_deviation"],
+        na=False,
```

Comment:
**[Question, non-blocking]** Am I right that this adds a `na` column to `df`? What's its purpose?

Reply:
Yes, good catch! Not sure why that is in there. I must have left it from older code.